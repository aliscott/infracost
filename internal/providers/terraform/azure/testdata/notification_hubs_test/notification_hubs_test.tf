provider "azurerm" {
  skip_provider_registration = true
  features {}
}

resource "azurerm_resource_group" "example" {
  name     = "example-notificationhub-resources"
  location = "eastus"
}

resource "azurerm_notification_hub_namespace" "example" {
  name                = "myappnamespace"
  resource_group_name = azurerm_resource_group.example.name
  location            = azurerm_resource_group.example.location
  namespace_type      = "NotificationHub"

  sku_name = "Basic"
}

resource "azurerm_notification_hub" "example" {
  name                = "mynotificationhub"
  namespace_name      = azurerm_notification_hub_namespace.example.name
  resource_group_name = azurerm_resource_group.example.name
  location            = azurerm_resource_group.example.location
}
