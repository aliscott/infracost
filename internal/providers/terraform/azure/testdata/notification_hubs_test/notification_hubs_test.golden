
 Name                                                 Monthly Qty  Unit                        Monthly Cost 
                                                                                                            
 azurerm_notification_hub_namespace.example                                                                 
 ├─ Base Charge Per Namespace (Basic)                           1  months                            $10.00 
 └─ Additional Pushes (over 10M)              Monthly cost depends on usage: $1.00 per 10 Million Pushes    
                                                                                                            
 azurerm_notification_hub_namespace.example1                                                                
 ├─ Base Charge Per Namespace (Standard)                        1  months                           $200.00 
 └─ Additional Pushes (over 10M)              Monthly cost depends on usage: $10.00 per 10 Million Pushes   
                                                                                                            
 PROJECT TOTAL                                                                                      $210.00 

----------------------------------
To estimate usage-based resources use --usage-file, see https://infracost.io/usage-file